Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    workos (2.8.0)
+    workos (2.8.1)
```

Comment:
Since this adds a new feature, I think this should be a minor release, `2.9.0` like how we did it in the other SDKs

Other than that looks good

Reply:
Yep, this should be version 2.9.0